charts/argo-events/templates/argo-events-webhook/deployment.yaml

```diff
@@ -37,7 +37,7 @@ spec:
       {{- end }}
       containers:
       - name: {{ .Values.webhook.name }}
-        image: {{ default .Values.global.image.repository .Values.webhook.image.repository }}:{{ default (include "argo-events.defaultTag" .) .Values.controller.image.tag }}
+        image: {{ default .Values.global.image.repository .Values.webhook.image.repository }}:{{ default (include "argo-events.defaultTag" .) .Values.webhook.image.tag }}
```

Comment:
^ WARNING: since there is a webhook.image.tag I'm assuming this was a bug so I'm providing a patch to this as well.

confirmed by @pdrastil on [this comment](https://github.com/argoproj/argo-helm/pull/1682/files#r1040876166).